README.md

```diff
@@ -114,19 +114,19 @@ For more usage details, see our detailed [documentation](#documentation).
 
 Benchmarks are run in GitHub Actions, and the tables below are updated with every code change. **Values >1.00x represent how much _slower_ each gem’s memoized value retrieval is than the latest commit of `MemoWise`**, according to [`benchmark-ips`](https://github.com/evanphx/benchmark-ips) (2.11.0).
 
-Results using Ruby 3.2.2:
+Results using Ruby 3.3.2:
 
 |Method arguments|`Dry::Core`\* (1.0.1)|`Memery` (1.5.0)|
 |--|--|--|
-|`()` (none)|0.66x|3.54x|
-|`(a)`|1.48x|8.49x|
-|`(a, b)`|1.18x|6.52x|
-|`(a:)`|1.53x|13.57x|
-|`(a:, b:)`|1.27x|10.56x|
-|`(a, b:)`|1.26x|10.44x|
-|`(a, *args)`|0.78x|1.60x|
-|`(a:, **kwargs)`|0.77x|2.12x|
-|`(a, *args, b:, **kwargs)`|0.69x|1.40x|
+|`()` (none)|0.60x|3.17x|
```

Comment:
I'm pretty surprised by the performance degradation here—worse results basically across the board. I don't have time to investigate though.